File: game/src/main/java/com/matag/game/launcher/LauncherGameResponseBuilder.java
package com.matag.game.launcher;

import org.springframework.stereotype.Component;

@Component
public class LauncherGameResponseBuilder {
  public String build(String session) {
    return
      "<html>\n" +
      "    <head>\n" +
      "        <meta charset=\"utf-8\">\n" +
      "        <title>MATAG - Game</title>\n" +
      "        <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n" +
      "        <link rel=\"shortcut icon\" type=\"image/png\" href=\"/img/matag.png\"/>\n" +
      "        <script>sessionStorage.setItem('token', '" + session + "')</script>\n" +
      "    </head>\n" +
      "    <body>\n" +
      "        <div id=\"app\" ></div>\n" +
      "        <script src=\"/js/game.js\" type=\"text/javascript\"></script>\n" +
      "    </body>\n" +
      "</html>";
  }
}

File: src/main/java/com/matag/game/launcher/LauncherGameResponseBuilder.kt
package com.matag.game.launcher

import org.springframework.stereotype.Component

@Component
class LauncherGameResponseBuilder {
    fun build(session: String) =
        """
            <html>
                <head>
                    <meta charset="utf-8">
                    <title>MATAG - Game</title>
                    <meta name="viewport" content="width=device-width, initial-scale=1.0">
                    <link rel="shortcut icon" type="image/png" href="/matag/game/img/matag.png"/>
                    <script>sessionStorage.setItem('token', '$session')</script>
                </head>
                <body>
                    <div id="app"></div>
                    <script src="/matag/game/js/game.js" type="text/javascript"></script>
                </body>
            </html>
        """.trimIndent()
}
